caf-audit-monkey-container/pom.xml

```diff
@@ -47,10 +47,6 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-slf4j-impl</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-api</artifactId>
-        </dependency>
```

Comment:
Is this change for this Jira?

Reply:
there was duplicity in the dependency import and the local build failed bcoz of it. So removed the duplicate entry.